From 7391e90a222dbf69ff49de468d069c9123a846ae Mon Sep 17 00:00:00 2001
From: Karl Williamson <khw@cpan.org>
Date: Mon, 4 Nov 2019 21:55:53 -0700
Subject: [PATCH] toke.c: comment changes
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

These should have been included in
0c311b7c345769239f38d0139ea7738feec5ca4d

Petr Písař: Ported to 5.28.2 from
d7f7b0e39a10a6e3e0bd81d15473ee522a064016.

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 toke.c | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/toke.c b/toke.c
index bd44d86..a1e6edf 100644
--- a/toke.c
+++ b/toke.c
@@ -3019,13 +3019,8 @@ S_scan_const(pTHX_ char *start)
                     s++;    /* Skip past the hyphen */
 
                     /* d now points to where the end-range character will be
-                     * placed.  Save it so won't have to go finding it later,
-                     * and drop down to get that character.  (Actually we
-                     * instead save the offset, to handle the case where a
-                     * realloc in the meantime could change the actual
-                     * pointer).  We'll finish processing the range the next
-                     * time through the loop */
-                    offset_to_max = d - SvPVX_const(sv);
+                     * placed.  Drop down to get that character.  We'll finish
+                     * processing the range the next time through the loop */
 
                     if (this_utf8 && UTF8_IS_ABOVE_LATIN1(*s)) {
                         has_above_latin1 = TRUE;
@@ -3042,8 +3037,6 @@ S_scan_const(pTHX_ char *start)
                  *      are the range start and range end, in order.
                  * 'd'  points to just beyond the range end in the 'sv' string,
                  *      where we would next place something
-                 * 'offset_to_max' is the offset in 'sv' at which the character
-                 *      (the range's maximum end point) before 'd'  begins.
                  */
                 char * max_ptr;
                 char * min_ptr;
-- 
2.21.0

